docs/2.drivers/netlify.md

```diff
@@ -43,7 +57,7 @@ To use, you will need to install `@netlify/blobs` as dependency or devDependency
 ```json
 {
   "devDependencies": {
-    "@netlify/blobs": "*"
+    "@netlify/blobs": "latest"
```

Comment:
Just checking, you would be okay opting users to use any latest major instead of documenting the tested latest major like ^1?

Reply:
We want them to use >=7 because of changes in the way keys are namespaced. I'll add a section explaining this.